docs/src/modules/components/AppFrame.js

```diff
@@ -136,6 +147,18 @@ class AppFrame extends React.Component {
     this.canonical = canonical;
   }
 
+  handleKeyDown = event => {
```

Comment:
We shouldn't need any JavaScript for this feature. I think that we can remove this code.

Reply:
Yup, that was one of my many attempts to mitigate the `scrollIntoView()` problem.